MiniApp/Classes/core/MiniApp.swift

```diff
@@ -1,7 +1,7 @@
 /// Mini App Public API methods
 public class MiniApp: NSObject {
     private static let shared = MiniApp()
-    private let realMiniApp = RealMiniApp()
+    let realMiniApp = RealMiniApp()
```

Comment:
Any reason? I see the build getting through even with private.

Reply:
it was for some tests I made and did not need in the end, I'll fix this